imports/ui/components/password-input.test.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-env mocha */
+
+import React from 'react'
+import { mount } from 'enzyme'
+import { expect } from 'meteor/practicalmeteor:chai'
+import PasswordInput from './password-input.jsx'
+import { Meteor } from 'meteor/meteor'
+
+if (Meteor.isClient) {
+  describe('PasswordInput', () => {
+    let passInp
+    beforeEach(() => {
+      passInp = mount(<PasswordInput inpRef={() => {}} />)
```

Comment:
Perhaps related to above... what is `inpRef`? (Could not find anything through DDG search.)

Reply:
inpRef is the function that's assigned to the "ref" property of the contained input. It allows the user of the component to bind a variable to the input's value